api/config/cdr_versions_test.json

```diff
@@ -9,6 +9,7 @@
     "releaseNumber": 1,
     "numParticipants": 946237,
     "cdrDbName": "cdr20181213",
+    "elasticIndexBaseName": "synth_r_2019q1_2",
     "publicDbName": "public20181213"
```

Comment:
Do we need this property anymore since databrowser is separate? Prob a change for a different PR.

Reply:
Right, will no longer be needed. There is a ticket near the top of the backlog for tearing down databrowser code